### PR TITLE
Tasks/#509 gitsynch on windows

### PR DESCRIPTION
**Pull request by hoavt**
connect #509 

@datphan please review

### CODE REVIEW

**Comment by datphan** [comment]
@hoavt pls request merge to `develop` branch, you are requesting merge to `master`

**Review by datphan — DISMISSED** [review]

File: docs/getting_started.rst [hunk 1]
```diff
@@ -363,5 +363,14 @@ https://help.github.com/articles/connecting-to-github-with-ssh/
    .. note::
 
       On Windows, you must always use ``Cygwin Terminal``, not ``Git Bash``.
+      If you have SSH configurated and vagrant is still not working on Windows, set the
```
Comment (hoatle) [inline docs/getting_started.rst]: configured

https://english.stackexchange.com/questions/416611/using-configurated-vs-configured

**Comment by hoatle** [comment]
make sure it works on both windows 7 and windows 10 at least

**Comment by hoavt** [comment]
> make sure it works on both windows 7 and windows 10 at least

Checked both

**Review by datphan — APPROVED** [review]

**Comment by hoatle** [comment]
```
hoatle [4:39 AM]
thấy PR https://github.com/teracyhq/dev/pull/550 không khớp với comment này của @datphan https://github.com/teracyhq/dev/issues/509#issuecomment-433502587
theo PR đó thì @datphan chạy được hay theo comment của em mới chạy được?


datphan [4:41 AM]
Cmt của e chạy trên máy e đc, chị Hoa thì k cần
Do cách setup ssh khác nhau nên thế

hoatle [4:42 AM]
ok e, vậy để a merge
```

so merging now...

**Review by hoatle — APPROVED** [review]
LGTM

**Pull request merged**